api/src/test/java/io/strimzi/api/kafka/model/KafkaMirrorMaker2Test.java

```diff
@@ -0,0 +1,17 @@
+/*
+ * Copyright Strimzi authors.
+ * License: Apache License 2.0 (see the file LICENSE or http://apache.org/licenses/LICENSE-2.0.html).
+ */
+package io.strimzi.api.kafka.model;
+
+/**
+ * The purpose of this test is to ensure:
+ *
+ * 1. we get a correct tree of POJOs when reading a JSON/YAML `KafkaMirrorMaker2` resource.
```

Comment:
Is there an item 2 to this list?

Reply:
Nope - the exact same javadoc is in `KafkaTest`, `KafkaConnectTest`, `KafkaTopicTest`, `KafkaUserTest`, etc. I've updated this one but left the others as they are for a subsequent PR.